ext/fetch/lib.rs

```diff
@@ -607,6 +607,7 @@ pub fn create_http_client(
   let mut builder = Client::builder()
     .redirect(Policy::none())
     .default_headers(headers)
+    .pool_max_idle_per_host(0)
```

Comment:
@bartlomieju tests are flaky (`connection closed before message completed`) unless this line is set. Of course, I only want this to be set just for running WPT. What would be the best way to achieve that? 

ref: https://github.com/hyperium/hyper/issues/2136

Reply:
@marcosc90 we have a `enable-testing-features-do-not-use` flag in cli we use for wpt tests